mirroroperator/registrymirror.py

```diff
@@ -43,9 +55,40 @@ def __init__(self, event_type, namespace, hostess_docker_registry,
             {},
         )
 
+        self.nginx_config_template = '''
+        proxy_cache_path {cache_dir} levels=1:2 inactive=7d use_temp_path=off keys_zone={zone}:10m;
+        server {{{{
+
+            listen                5000 ssl;
+            server_name           localhost;
+            ssl_certificate       {registry_cert_dir}/tls.crt;
+            ssl_certificate_key   {registry_cert_dir}/tls.key;
+
+            location {healthcheck_path} {{{{
```

Comment:
Previously we were using `GET /` as a readiness check, but there is no guarantee that on the upstream `/` will necessarily return 200 status. Furthermore, we want the mirror to stay up and serve cached images even if the upstream is down.